build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -32,7 +32,15 @@
         <Link>app.config</Link>
       </None>
     </ItemGroup>
+  </Target>
 
+  <!-- Add Test Explorer Service GUID required by VS, and error out if it was specified in the project file -->
```

Comment:
Yes. I want it to be added after the check.

---
In reply to: [103039890](https://github.com/dotnet/roslyn/pull/17370#discussion_r103039890) [](ancestors = 103039890)